docs/java/queries.md

```diff
@@ -3,3 +3,53 @@ id: queries
 title: Queries in Java
 sidebar_label: Queries
 ---
+
+Temporal provides a query feature that supports synchronously returning any information from a Workflow to an external caller.
+
+Example Workflow interface that can respond to Queries:
+
+```java
+  @WorkflowInterface
+  public interface HelloWorld {
+    @WorkflowMethod
+    void sayHello(String name);
+
+    @QueryMethod
+    int getCount();
+  }
+```
+
```

Comment:
Similar to the activities section, QueryMethod also has a "name" parameter. We should maybe describe it as well

Reply:
added